.github/workflows/test.yml

```diff
@@ -175,16 +180,31 @@ jobs:
         with:
           image: "docker-compose_workflow-service:latest"
 
+      - name: Download images
+        uses: ishworkh/docker-image-artifact-download@v1
+        with:
+          image: "docker-compose_workflow-examples:latest"
+
       - name: Load images inside kind
         run: |
           make push-images-to-kind
 
+      - name: Copy workflow-examples to PVC
+        run: |
+          kubectl create -f hack/manifests/testing/copy-to-pvc.yaml
```

Comment:
@eloycoto at this point, do we have the workflow jar available in the downloaded artifacts?

if we have it, we can replace packaging the workflow-jar as image and instead of using pod to copy form one pod to another PVC with `kubectl cp` option
https://www.middlewareinventory.com/blog/kubectl-cp-example/

Reply:
@masayag I tested it. it works. I don't think that an image is a problem here. you will still need the extra pod. Because 'kubectl cp' only copies to a container. but with an image it is less intrusive to the way pods work. compared to 'kubectl cp'. and you get all the logic in the pod yaml instead of a pod and a 'kubectl cp' command.

Reply:
notice that the image is only used locally. it is not pushed to Quay.